pest_bootstrap doesn't build

```
➜  pest git:(master) cargo bootstrap
   Compiling pest_bootstrap v0.0.0
error[E0308]: mismatched types
  --> bootstrap/src/main.rs:24:23
   |
24 |         derive_parser(pest, false)
   |                       ^^^^ expected struct `proc_macro2::TokenStream`, found struct `quote::__rt::TokenStream`
   |
   = note: expected type `proc_macro2::TokenStream`
              found type `quote::__rt::TokenStream`
note: Perhaps two different versions of crate `proc_macro2` are being used?
  --> bootstrap/src/main.rs:24:23
   |
24 |         derive_parser(pest, false)
   |                       ^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: Could not compile `pest_bootstrap`.

To learn more, run the command again with --verbose.
```

Versions of tools that I use:
```
➜  pest git:(master) rustup toolchain list
stable-x86_64-apple-darwin (default)
nightly-x86_64-apple-darwin
➜  pest git:(master) rustc --version
rustc 1.38.0 (625451e37 2019-09-23)
➜  pest git:(master) rustc +nightly --version
rustc 1.40.0-nightly (ddf43867a 2019-09-26)
```

I'm not entirely sure why this happened, despite the fact that it also fails to build on CI. Are there any specific version requirements?

--- a/bootstrap/Cargo.toml
+++ b/bootstrap/Cargo.toml
@@ -10,8 +10,8 @@ publish = false
 license = "MIT/Apache-2.0"
 
 [dependencies]
-pest_generator = "2.1.0" # Use the crates-io version, which (should be) known-good
-quote = "0.6.8"
+pest_generator = "2.1.1" # Use the crates-io version, which (should be) known-good
+quote = "1.0"
 
 [badges]
 codecov = { repository = "pest-parser/pest" }

